Description: Hard-code paths to system shared data
 This could be made generic and forwarded upstream. For example, add another
 SAGE_SHARE_DATA directory, which for upstream would be the same as SAGE_SHARE
 and for us would be /usr/share/sagemeth. However, this may not be worth the
 effort:
 - upstream already tried it before (as SAGE_DATA) and decide to ditch that
 - for upstream to accept it again, we'd also have to patch lots of things that
   are pointless for Debian, such as the scripts in build/pkgs
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO
--- a/sage/src/sage/combinat/designs/latin_squares.py
+++ b/sage/src/sage/combinat/designs/latin_squares.py
@@ -547,8 +547,7 @@
         return
 
     if compare:
-        from sage.env import SAGE_SHARE
-        handbook_file = open(SAGE_SHARE+"/combinatorial_designs/MOLS_table.txt",'r')
+        handbook_file = open("/usr/share/sagemath/combinatorial_designs/MOLS_table.txt",'r')
         hb = [int(_) for _ in handbook_file.readlines()[9].split(',')]
         handbook_file.close()
 
--- a/sage/src/sage/databases/cremona.py
+++ b/sage/src/sage/databases/cremona.py
@@ -54,12 +54,13 @@
 import sage.schemes.elliptic_curves.constructor as elliptic
 from .sql_db import SQLDatabase, verify_column
 from sage.misc.package import is_package_installed
-from sage.env import SAGE_SHARE
 from sage.misc.all import walltime
 
 import re
 import string
 
+SAGE_SHARE = '/usr/share/sagemath/'
+
 _cremonaSkeleton = {
     't_class': {
         'conductor': {'sql':'INTEGER', 'index':True},
@@ -1663,8 +1664,7 @@
         Traceback (most recent call last):
         ...
         ValueError: Desired database (='should not exist') does not exist
-        sage: from sage.env import SAGE_SHARE
-        sage: os.path.isfile(os.path.join(SAGE_SHARE,'cremona','should_not_exist.db'))
+        sage: os.path.isfile(os.path.join('/usr/share/sagemath/cremona','should_not_exist.db'))
         False
     """
     global _db
--- a/sage/src/sage/graphs/strongly_regular_db.pyx
+++ b/sage/src/sage/graphs/strongly_regular_db.pyx
@@ -3213,8 +3213,7 @@
         return
     import json
 
-    from sage.env import SAGE_SHARE
-    with open(SAGE_SHARE+"/graphs/brouwer_srg_database.json",'r') as datafile:
+    with open("/usr/share/sagemath/graphs/brouwer_srg_database.json",'r') as datafile:
         _brouwer_database = {(v,k,l,mu):{'status':status,'comments':comments}
                              for (v,k,l,mu,status,comments) in json.load(datafile)}
 
